From d72d2f83236d6dae7759fd986527c8b88be27762 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@nxp.com>
Date: Tue, 20 Nov 2018 14:49:45 +0800
Subject: [PATCH 5156/5242] MLK-20417-1: ARM64: dts: disable intmux in m4 dts

commit  5441cb25be18961d73db4801c3213912144c9874 from
https://source.codeaurora.org/external/imx/linux-imx.git

disable intmux in m4 dts for it is component in m4 domain

Signed-off-by: Shengjiu Wang <shengjiu.wang@nxp.com>
(cherry picked from commit 2742df294bac6e1ccc1ed7e899c0b5c85a51ccbf)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qxp-mek-m4.dts      |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-m4.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-m4.dts
index 63c9a00..2f9d9fdb 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-m4.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-m4.dts
@@ -96,6 +96,10 @@
 	status = "disabled";
 };
 
+&intmux_cm40 {
+	status = "disabled";
+};
+
 &flexcan1 {
 	status = "disabled";
 };
-- 
1.7.9.5

